CHANGES/6410.removal

```diff
@@ -0,0 +1 @@
+Remove PackageGroup, PackageCategory and PackageEnvironment relations to packages, to each other and its generation from sync.
```

Comment:
I'd omit the last part because that's an implementation detail.
How about just  `Remove PackageGroup, PackageCategory and PackageEnvironment relations to packages and to each other` ?